ember-toucan-core/src/components/form/input-field.hbs

```diff
@@ -9,7 +9,7 @@
       <field.Hint id={{field.hintId}} data-hint>{{@hint}}</field.Hint>
     {{/if}}
 
-    <field.Control class="flex">
+    <field.Control class="mt-1.5 flex">
```

Comment:
It's tempting to add `mt-1.5` directly to `field.Control`, but we have cases where that'll actually cause unwanted padding like with Checkbox, RadioField, etc.